Sources/xcprojectlint-package/Validation.swift

```diff
@@ -57,12 +61,13 @@ List of validations to perform:
                      all:
                        Runs all known validations
 """
-  
-  public static func allValidations() -> [Validation] {
-    return [Validation.buildSettingsExternalized,
-            Validation.diskLayoutMatchesProject,
-            Validation.filesExistOnDisk,
-            Validation.itemsInAlphaOrder,
-            Validation.noEmptyGroups]
-  }
+
+    public static func allValidations() -> [Validation] {
+        return [Validation.buildSettingsExternalized,
+                Validation.diskLayoutMatchesProject,
+                Validation.filesExistOnDisk,
+                Validation.itemsInAlphaOrder,
+                Validation.noDanglingSourceFiles,
+                Validation.noEmptyGroups]
+    }
```

Comment:
would be so much easier to review this file without the noise of the formatting changes :-(

Reply:
Agreed. I think we need to do some history re-writing and remove all the formatting changes before proceeding.